hack/e2e-test.sh

```diff
@@ -136,9 +135,7 @@ while [ $i -le ${max_tries} ]; do
   fi
 
   echo "Applying ClusterDeployment File ${CLUSTER_NAME}. Try #${i}/${max_tries}:"
-  if oc apply -f ${CLUSTER_DEPLOYMENT_FILE} ; then
-    echo "Success"
-    break
+  if oc apply -f ${CLUSTER_DEPLOYMENT_FILE} ; then echo "Success" break
```

Comment:
@dgoodwin you need a `;` between `echo "Success"` and `break`
(the e2e test kept repeating this apply 60 times)

Reply:
This looks like a vim keypress screwup, I have no memory of making this change intentionally.